linters/bandit.js

```diff
@@ -53,7 +53,24 @@ module.exports = class Bandit {
    * @param {Buffer} data The raw linter results data
    */
   parseResults (data) {
-    return JSON.parse(data)
+    let parsedData = JSON.parse(data)
+
+    if (parsedData.errors.length > 0) {
+      for (let i = 0; i < parsedData.errors.length; ++i) {
+        let errAnnotation = {
+          filename: parsedData.errors[i].filename,
+          line_number: 1,
+          issue_severity: 'HIGH',
+          issue_confidence: 'HIGH',
+          issue_text: '',
+          test_id: 'ERROR',
+          test_name: 'Syntax errors' }
```

Comment:
errors is plural indicating more than one.  we don't know for sure there are multiple syntax errors.